app/views/simulator/edit.html.erb

```diff
@@ -22,14 +22,14 @@
   projectName = "<%= @projectName %>"
 </script>
 <div id='exitView'></div>
-<nav class="navbar navbar-expand-lg navbar-dark header">
```

Comment:
`header` is a CSS class you cannot remove. It is referenced in `app/javascript/src/sass/color_theme.scss`

Reply:
Where is `bg-dark` defined?

Reply:
@satu0king, thanks for the review
  header class also defines a property of height:50px in main.stylesheet.css which does not allow  navbar to collapse that's why I removed header class
also header defines color property as you said 
bg-dark defines colour #343a40 and header defines colour #454545
I prefered bg-dark because it is possible that .header may be used somwhere at other places
and removing height:50px defined in .header  from main.stylesheet.css can effect other components

so we can do is define height of navbar only for 992px and greater 
 but below 992px and less height is automatically adjusted  so this will give a inconsistent look 

If .header is not used at any other place  we can remove height:50 px property and nav will have colour #454545 and will be in sync with colours of other components

suggest me! ,according  to it i will make changes...

Reply:
The color of the header is set by color themes, so you should not be setting the color. Please remove `bg-dark`. I am curious as to where is `bg-dark` defined. Anyway, as far as I can see `header` is not being used anywhere. @abstrekt can confirm the same. So please make changes to the `header` defined in the main simulator stylesheets file.

Reply:
bg-dark is most probably a bootstrap class, I remember removing it somehow it came through, it is not needed,
 main.stylesheet.css is only attached to edit.html.erb, I mean only attached to the simulator, so it won't affect any other components, better to make changes to existing classes if possible is better

Reply:
ok great !  then there is no prblm in removing height property